src/client/components/Forms/ChangePasswordForm.tsx

```diff
@@ -15,7 +15,7 @@ export const ChangePasswordForm = () => {
   const repeatNewPasswordRef = React.useRef<HTMLInputElement>(null);
   const { data: session, status } = useSession();
 
-  const [error, setError] = React.useState();
+  const [error, setError] = React.useState(2);
```

Comment:
Why there is an error initially?

Reply:
It was a leftover from debugging that I forgot to clean up